### ISSUE TITLE
Track clicks with Google Analytics

### ISSUE THREAD

**Issue by rafenden** [thedevs-network/kutt#123]
Great piece of software!

Have you considered tracking clicks with Google Analytics additionally to the internal tracking?

This functionality could be optional and added to the `createVisit()` function with `universal-analytics` NPM package.

**Comment by poeti8** [comment]
Yes, I was thinking about this recently. It's a good idea, I'm gonna place it on a high priority.

**Comment by rafenden** [comment]
Was this feature implemented or rejected?

**Comment by poeti8** [comment]
This feature has been implemented for the owner of the website. I'm going to add a field in settings page soon so each user can set Google Analytics tracking for their links.
